tests/monotouch-test/MediaAccessibility/ImageCaptioningTest.cs

```diff
@@ -103,7 +104,11 @@ public void SetCaption ()
 					Assert.Null (e, "ro / set / no error"); // weird, it can't be saved back to the file metadata
 
 					var s = MAImageCaptioning.GetCaption (url, out e);
+#if __MACOS__
+					if (TestRuntime.CheckXcodeVersion (12, 2)) {
+#else
 					if (TestRuntime.CheckXcodeVersion (12, 0)) {
+#endif
```

Comment:
minor: since this happens a few time in the same file you could have a single
```
#if __MACOS__
const int MinorXcode11Mismatch = 2;
#else
const int MinorXcode11Mismatch = 0;
#endif
```
and use that constant in the checks (no more `#ifdef` for that file).

It could also be added into `TestRuntime` so it would be usable in all files.

Reply:
Oh, I like this better it keeps the intent and truth, will re-work on top of this idea 👍

Reply:
Apple 😡

Adding something like `TestRuntime.CheckApiAddedInSummer2020 ()` could work (might need a better name...), and that would avoid the #ifs too.